### ISSUE TITLE
[Feature Request] Choose Category Expansion Parameters

### ISSUE THREAD

**Issue by someONEgood** [UnknownX7/QoLBar#31]
First of all, **THANK YOU FOR THIS WONDERFUL PLUGIN**. It's such a wonderful framework on which to put the various plugins the talented community has made. You're awesome.

As for the feature request, I'm not certain if it's my lack of knowledge with the plugin, or if the feature is just not there.
I was wondering if it's possible to choose which side (above, beneath, sides) the category button opens up?
As well as whether the bar expands horizontally or vertically?
Basically was trying to make a job select bar that's nested:

![qol](https://user-images.githubusercontent.com/46905309/136855670-8320a70c-c7e3-4cb4-9d26-32def9f01a4e.JPG)

For instance clicking the tank button expands and brings up the WAR/PLD macros.
However I was wanting it to a) open to the right side and b) expand horizontally.
If the option to choose those variables could be implemented it would be really appreciated. Thank you again for your time and effort.

**Comment by UnknownX7** [comment]
Horizontal is just setting the columns of the category to 0, side however is automatically determined depending on which direction has more room (and if the bar is more horizontal or vertical). Many people have requested that though so I may allow overriding it soon.
